### DISCUSSION TITLE
Help for Display of Thermal Hydraulic Module Calculation Results

### DISCUSSION THREAD

**Discussion by LGQ199810** [idaholab/moose#29031] [Q&A Modules: Thermal Hydraulics] [answered]
At present, I have used Paraview to read the calculation results of the circulation loop constructed based on the thermal hydraulic module. However, temperature cloud maps are based on line displays, which are too thin to distinguish. Is there any way to make the lines in the displayed temperature/pressure cloud maps thicker or more intuitive, or is there any tutorial to read the calculation results of thermal hydraulic modules based on Paraview? Thank you very much!
![Paraview_pic](https://github.com/user-attachments/assets/797cedae-2c8f-44d7-bc2a-b1bcb1888902)

**Comment by GiudGiud** [comment, marked as answer]
hello

in the settings in the Properties tab you can click the "cog" to show more parameters, then increase the line thichkness

**Reply by LGQ199810** [reply]
Wow, I found that option, it helped me a lot. Sincere thanks!
